packages/storybook-react/src/stories/ButtonLink/argTypes.ts

```diff
@@ -0,0 +1,10 @@
+export const argTypes = {
+  appearance: {
+    type: 'select',
+    options: ['primary-action-button', 'secondary-action-button', 'subtle-button'],
+  },
+  type: {
+    type: 'select',
+    options: ['button', 'submit', 'reset'],
+  },
```

Comment:
The `ButtonLink` component doesn't have a `type` argument (since it is not actually a `<button>`)

And the `external` property is missing from `argTypes` (since it is a link)